packages/create-cloudflare/e2e-tests/pages.test.ts

```diff
@@ -22,8 +34,117 @@ type FrameworkTestConfig = RunnerConfig & {
 };
 
 describe.concurrent(`E2E: Web frameworks`, () => {
+	// These are ordered based on speed and reliability for ease of debugging
+	const frameworkTests: Record<string, FrameworkTestConfig> = {
+		astro: {
+			expectResponseToContain: "Hello, Astronaut!",
+			testCommitMessage: true,
+		},
+		docusaurus: {
+			expectResponseToContain: "Dinosaurs are cool",
+			testCommitMessage: true,
+		},
+		gatsby: {
+			quarantine: true,
```

Comment:
does gatsby need to be quarantined? I thought you made it better 🤔

Reply:
It's better but not better enough quite yet. It's failing somewhat consistently with `bun` atm and I need to debug why. Don't worry, clearing out the quarantine is next on my list of things to do.

Reply:
Took it out of quarantine in another PR: https://github.com/cloudflare/workers-sdk/pull/4120/files